app/controllers/application_controller.rb

```diff
@@ -1,5 +1,16 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @user = User.first
+  protect_from_forgery with: :exception
+  # protect_from_forgery prepend: true
```

Comment:
You did great job, However, you need to fix some errors to make your project approve. 

- [ ] Kindly remove inline commented codes from your file :)

Reply:
@Rustamxon7 , as requested, all comments have been deleted